uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -43,6 +44,10 @@ define([
         expect($scope.role).toBe('admin');
       });
 
+      it('sets the modal title as "New Document" by default', function () {
```

Comment:
Instead of testing this here, test in `$scope.modalTitle()` describe block. To properly organise the unit tests.